core/cleaner/pom.xml

```diff
@@ -33,6 +33,22 @@
   <name>AET :: Core :: Cleaner</name>
   <description>Cleaner responsible for removing outdated AET artifacts from datastorage</description>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
```

Comment:
If those are dependencies only for tests then we should probably add`<scope>test</scope>`

Reply:
done :wink: